components/formats-bsd/src/loci/formats/in/MicromanagerReader.java

```diff
@@ -626,11 +626,11 @@ else if (token.trim().startsWith("}")) {
 
           if (key.equals("Exposure-ms")) {
             double t = Double.parseDouble(value);
-            p.exposureTime = new Time(new Double(t / 1000), UNITS.S);
+            p.exposureTime = new Time(new Double(t), UNITS.MS);
           }
           else if (key.equals("ElapsedTime-ms")) {
             double t = Double.parseDouble(value);
-            stamps.add(new Double(t / 1000));
+            stamps.add(new Double(t));
```

Comment:
This (and exposureTime above) could probably be simplified to just one line e.g. ```stamps.add(Double.valueOf(value))```.